### ISSUE TITLE
No spaces in directory (or file) names

### ISSUE THREAD

**Issue by craigsapp** [music-encoding/sample-encodings#25]
It would be useful to not uses spaces in directory names.   For example, `sample-encodings/MEI 3.0` should be renamed to `sample-encodings/MEI-3.0`.  Notice that `sample-encodings` avoids the space, but `MEI 3.0` does not, so there is currently some inconsistency in the naming of directories.

This is also true of the `Musical features` directory:  `sample-encodings/MEI 3.0/Musical features/snippets` which should instead be `sample-encodings/MEI-3.0/Musical-features/snippets` and preferably make the "M" lower case:  `sample-encodings/MEI-3.0/musical-features/snippets` (less important for MEI, although `mei` is somewhat better as well).

In other words, file/directory names should be POSIX compliant for cross-platform compatibility, and that means only use these characters in filenames:

```
ABCDEFGHIJKLMNOPQRSTUVWXYZ
abcdefghijklmnopqrstuvwxyz
0123456789._-
```

From section 3.276 (Portable Filename Character Set) of the Open Group Base Specification:
     http://opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_276

**Comment by pe-ro** [comment]
Closed by 315c1c7

**Comment by craigsapp** [comment]
Looks great!
